cmake/modules/SwiftSupport.cmake

```diff
@@ -9,8 +9,12 @@
 function(get_swift_host_arch result_var_name)
   if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
     set("${result_var_name}" "x86_64" PARENT_SCOPE)
-  elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64")
-    set("${result_var_name}" "aarch64" PARENT_SCOPE)
+  elseif ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "AArch64|aarch64|arm64")
```

Comment:
Similar to swift-system, not sure if the `AArch64` case is needed, but this seems fine.

Reply:
I can delete `AArch64 ` if not needed. the code is just copied from swift-atomics/cmake/modules/SwiftSupport.cmake. Better to keep it

Reply:
I think both are needed to cover apple silicon and arm linux. cc @drexin

Reply:
@tomerd I'm not sure - I'm talking about _literally_ `AArch64`, not `aarch64`.  Its a question of is there an extra case being matched against or not.  Either way, it doesn't actually matter,

Reply:
I’m not sure about this particular case, but I have seen AMD64 and amd64 on different platforms. I don’t think it hurts to keep it as is.